sw/airborne/modules/computer_vision/video_thread.c

```diff
@@ -122,6 +122,10 @@ static void *video_thread_function(void *data)
                 vid->w, vid->h, vid->fps, 1000000.f / dt_us);
       }
     }
+    FloatEulers imageStartEulers;
+    imageStartEulers.phi=stateGetNedToBodyEulers_f()->phi;
```

Comment:
You need to protect those variables to make it thread safe..
Also it is nicer to fetch the eulers with `stateGetNedToBodyEulers_f()` only once. Next to that I guess you could add it to the `image_t` as a parameter instead of al those seperate variables.